lib/send_sms.rb

```diff
@@ -0,0 +1,29 @@
+require 'twilio-ruby'
+
+class Send_SMS
+
+  def confirmation_text(args)
+    sender(args)
+  end
+
+
+  private
+
+  def credentials(value)
+    return 'AC3c30852abaf447e0f8c4b51eb23b0d01' if value == :sid
+    return '5c5c4e416d9ac5b15ce57e0f12eb6d37' if value == :token
+    return '+441477652033' if value == :from
+  end
+
+  def sender(args)
+    client = Twilio::REST::Client.new credentials(:sid), credentials(:token)
+    client.account.messages.create(from: credentials(:from), to: args[:phone],
+    body: "Thank you! #{args[:name]}, your order was placed and will be delivered before #{stamp_time}")
```

Comment:
Metrics/LineLength: Line is too long. [104/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)